tests/internships_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from unittest.mock import MagicMock, patch
+
+from data_collections.internships import getInternships
```

Comment:
Update the module path:
```suggestion
from data_collections.rss_parser import getInternships
```